extra-fonts/ttf-misaki/spec

```diff
@@ -0,0 +1,4 @@
+_URL_VER=2019-10-19
+VER=${_URL_VER//-/}
```

Comment:
Wrong version styling - this should have been 2019.10.19. You could have defined `VER=` as 2019.10.19, and used `${VER//./-}` below.